lib/improver/tests/utilities/cube_manipulation/test_merge_cubes.py

```diff
@@ -120,18 +137,71 @@ def test_lagged_ukv(self):
     def test_multi_model(self):
         """Test Multi models merge OK"""
         cubes = iris.cube.CubeList([self.cube, self.cube_ukv])
-        result = merge_cubes(cubes)
+        result = merge_cubes(cubes, "mosg__model_configuration")
```

Comment:
merge_cubes takes a keyword argument, so ideally we would use the keyword in the call:

```suggestion
        result = merge_cubes(cubes, model_id_attr="mosg__model_configuration")
```

Reply:
fixed